### PR TITLE
Add zoneinfo to the windows bundle

### LINKED ISSUE #689: b2 2.1.0 Windows version executable is missing dateutil-zoneinfo package
Using latest b2-windows executable on Window 10.0.18363.1379:

    >b2-windows.exe version
    b2 command line tool, version 2.1.0
    >b2-windows.exe authorize-account <account> <master_key>
    Using https://api.backblazeb2.com
    dateutil\zoneinfo\__init__.py:26: UserWarning: I/O error(2): No such file or directory

Checking with Process Monitor shows that embedded python is looking for this file:

    dateutil\zoneinfo\dateutil-zoneinfo.tar.gz

Tool seems to still work, so the issue is probably cosmetic.

### PR DESCRIPTION
**Pull request by mlech-reef**
Closes #689

### CODE REVIEW

**Review by mpnowacki-reef — APPROVED** [review]

File: test/integration/test_b2_command_line.py [hunk 1]
```diff
@@ -133,8 +133,7 @@ def read_from(self, f):
 
 def remove_warnings(text):
     return os.linesep.join(
-        line for line in text.split(os.linesep)
-        if 'DeprecationWarning' not in line and 'UserWarning' not in line
+        line for line in text.split(os.linesep) if 'DeprecationWarning' not in line
```
Comment (mpnowacki-reef) [inline test/integration/test_b2_command_line.py:136]: why this change?

Comment (mlech-reef) [reply test/integration/test_b2_command_line.py:136]: Missing zoneinfo caused `UserWarning`. We don't need this anymore
